BJS2-84251-ci_pipeline

Linempy opened the pull request:
Задачка "CI pipeline для project_service"

Linempy committed:
BJS2-84251-ci_pipeline
- добавил ci-pipeline

RinatBeybutov commented on .github/workflow/ci-prs.yaml:
```diff
@@ -0,0 +1,55 @@
+name: Check PRs
```
CI pipeline manticore11

RinatBeybutov commented on .github/workflow/ci-prs.yaml:
```diff
@@ -0,0 +1,55 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: check repository
```
- name: check repository -> - name: STEP_1 - check repository
и далее тоже

Linempy committed:
BJS2-84251-ci_pipeline
- поправил ci pipeline
- добавил в pipeline проверку на checkStyle
- исправил код по checkstyle

RinatBeybutov commented:
вмержи изменения из мастера и мержи эту ветку

Linempy committed:
Merge branch 'manticore-master-stream11' into manticore11/BJS2-84251-ci_pipeline
